project/Prototypes.scala

```diff
@@ -36,7 +36,8 @@ trait Prototypes {
     cleanAll := Def.taskDyn {
       val allProjects = ScopeFilter(inAnyProject)
       clean.all(allProjects)
-    }.value
+    }.value,
+    unmanagedClasspath in Runtime += file("ui/dist")
```

Comment:
This line tells sbt to include the`ui/dist` directory into the classpath, so its content is available.

I agree with you that failing at compilation if/when the javascript bundle is not present (or even better when it doesn't compile) is the better solution but I figured we could do without it for now. It is on my list to add asap though.

Can you show me the current behaviour with the client-side assets you are describing in the comment? Thank you